recipe/meta.yaml

```
@@ -28,17 +28,20 @@ requirements:
     - blinker >=1.0.0,<2
     - cachetools >=4.0,<6
     - click >=7.0,<9
-    - numpy >=1.20,<3
-    - packaging >=20,<25
+    - importlib-metadata >=1.4,<7
+    - numpy >=1.19.3,<3
```

```suggestion
    - numpy >=1.19.3,<2
```

https://github.com/streamlit/streamlit/blob/1.29.0/lib/setup.py#L38